docs/docsite/rst/conf.py

```diff
@@ -175,6 +175,10 @@
     'porting_guides/core_porting_guides',
 ] if tags.has('ansible') else '<UNKNOWN>'
 
+nitpick_ignore_regex = [
+    (r"py:.*", r"ansible\..*_internal\..*"),
```

Comment:
Should there be another dot before `_internal`?
```suggestion
    (r"py:.*", r"ansible\..*\._internal\..*"),
```
That way, `ansible.something_internal.foo` is not matched, only `ansible.something._internal.foo`.

Reply:
Oh, and maybe start the regex with `^`?

Reply:
This way it matches `ansible._internal` too.

Reply:
Ah right. But
```suggestion
    (r"py:.*", r"ansible\.(?:.*\.)?_internal\..*"),
```
would do that too :)

Reply:
Thanks for the suggestion, fixed.